apps/widget/src/app/ModelAssessment.tsx

```diff
@@ -32,6 +32,8 @@ export class ModelAssessment extends React.Component<IModelAssessmentProps> {
       | "requestImportances"
       | "requestCausalWhatIf"
       | "requestBoxPlotDistribution"
+      | "requestDatasetAnalysisBarChart"
```

Comment:
This is becoming a rather massive list. Should we bundle these together somehow? Otherwise we'll end up with a hundred input params. Thoughts?

Reply:
Created.